examples/web-demo/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
Is this dead code or some kind of compiler directive?

Reply:
I believe it's the latter; it's auto-generated